packages/eslint-plugin-choreography-ts/src/no-renaming-operator.ts

```diff
@@ -0,0 +1,65 @@
+// no-renaming-operator.js: must NOT rename choreographic operators
+// https://github.com/shumbo/choreography-ts/issues/19
+
+// Create custom typescript rules: https://typescript-eslint.io/developers/custom-rules/
+// Better: https://medium.com/inato/using-typescript-to-build-custom-eslint-rules-faster-53ad1c9dee2b
+"use strict";
+import { AST_NODE_TYPES, TSESTree, TSESLint } from "@typescript-eslint/utils";
+
+type MessageIDs = "rename" | "invalid";
+
+const noRenameRule: TSESLint.RuleModule<MessageIDs, []> = {
```

Comment:
The typescript-eslint document says the recommended way of defining rules is to use `RuleCreator`.

https://typescript-eslint.io/developers/custom-rules#rulecreator

Do you know the difference between using this vs. creating a rule object manually, as you did in this file?

Reply:
This is just what ended up working for me and is what was shown at this link: https://medium.com/inato/using-typescript-to-build-custom-eslint-rules-faster-53ad1c9dee2b
The recommended way also wasn't working for some reason, although I can trying doing that again if you prefer it

Reply:
OK. Let's keep this as is for now. It should be fairly easy to migrate if we want to.